src/app/api/routes/sites.py

```diff
@@ -71,11 +72,14 @@ async def fetch_sites(requester=Security(get_current_access,
 async def update_site(
     payload: SiteIn,
     site_id: int = Path(..., gt=0),
-    _=Security(get_current_access, scopes=[AccessType.admin])
+    requester=Security(get_current_access, scopes=[AccessType.admin, AccessType.user])
 ):
     """
     Based on a site_id, updates information about the specified site
     """
+    # TODO: validate this one
```

Comment:
validate the general design of group checking you mean?

Reply:
Validate that the route need to haveccess restrcted to group